ground/src/main/java/com/google/android/ground/model/mutation/SubmissionMutation.kt

```diff
@@ -19,7 +19,8 @@ import com.google.android.ground.model.job.Job
 import com.google.android.ground.model.submission.TaskDataDelta
 import java.util.*
 
-data class SubmissionMutation(
+data class SubmissionMutation
+constructor(
```

Comment:
Can we remove the `constructor` keyword for consistency with rest of the codebase?

Reply:
Done